compiler/ballerina-lang/src/main/java/io/ballerina/projects/util/ProjectConstants.java

```diff
@@ -111,4 +111,13 @@ public class ProjectConstants {
     public static final String LOCAL_REPOSITORY_NAME = "local";
     public static final String CENTRAL_REPOSITORY_CACHE_NAME = "central.ballerina.io";
     public static final String DEPENDENCIES_TOML_VERSION = "2";
+
+    // graphql-client template (`bal new <package_name> -t graphql-client`) specific constants
+    public static final String GRAPHQL_CONFIG_FILE_NAME = "graphql.config.yaml";
```

Comment:
Loos like these constants are declared for test cases. Shall we move these to the particular test file?